README.md

```diff
@@ -208,27 +180,224 @@ schema = infer(headers, values)
 
 The number of rows used by `infer` can be limited with the `row_limit` argument.
 
-### CLI
+### Validate
+
+Given a schema as JSON file, url to JSON file, or a Python dict, `validate` returns `True` for a valid JSON Table Schema, or raises an exception, `SchemaValidationError`.
+
+```python
+import io
+import json
+
+from jsontableschema import validate
+
+filepath = 'schema_to_validate.json'
+
+with io.open(filepath) as stream:
+    schema = json.load(stream)
+
+is_valid = jsontableschema.validate(schema)
```

Comment:
It's not a part of PR's piece of work but I think it's OK to fix it here - I've updated PR.